docs/quick_start.md

```diff
@@ -136,15 +114,15 @@ Honeycomb, like most applications, contains a template called `package.json`. Th
 
 ##### 1. Metadata
 
- ```json
+```json
 {
-  "name": "honeycomb",
-  "description": "all-in-one task starter app with jsPsych + React + Electron + psiturk ",
-  "author": {
-    "name": "Brown CCV",
-    "email": "ccv-bot@brown.edu",
-    "url": "ccv.brown.edu"
-  },
+ "name": "honeycomb",
+ "description": "all-in-one task starter app with jsPsych + React + Electron + psiturk ",
+ "author": {
+   "name": "Brown CCV",
+   "email": "ccv-bot@brown.edu",
+   "url": "ccv.brown.edu"
+ },
```

Comment:
```suggestion
 }
```

I think you need to take out this comma to make the formatting happy?

Reply:
Oh yes, was missing an additional comma too! All fixed 😄